fem/bilinearform.hpp

```diff
@@ -848,6 +848,12 @@ class MixedBilinearForm : public Matrix
         to it.  Used for transferring ownership. */
    SparseMatrix *LoseMat() { SparseMatrix *tmp = mat; mat = NULL; return tmp; }
 
+   /// Returns a const reference to the sparse matrix of eliminated b.c.:  $ M_e $
+   const SparseMatrix &SpMatElim() const { return *mat_e; }
```

Comment:
Should probably use the same sort of `MFEM_VERIFY` call used in `BilinearForm`.

Reply:
A good point, I added it also to `SpMat()`. 😉 .

Reply:
Perfect! Thank you.